src/com/xilinx/rapidwright/interchange/PhysNetlistWriter.java

```diff
@@ -455,6 +455,19 @@ private static void populateRouting(List<RouteBranchNode> routingBranches,
             sources = new ArrayList<>();
             stubs = new ArrayList<>();
 
+            // To be used in conjunction with the "rapidwright.rwroute.lutPinSwapping.deferIntraSiteRoutingUpdates"
+            // property (which allows RWRoute to perform LUT pin swapping but not move the swapped SitePinInst nor
+            // perform any intra-site routing updates). With the aforementioned RWRoute option, for swapped LUT pins
+            // default PhysNetlistWriter would not be able to identify the site pin that the routing services, thus
+            // causing any deferred site pin updates to appear as unconnected stubs.
+            // Enabling this following option allows the PhysNetlistWriter to simulate LUT pin swapping such that
+            // any routing servicing a LUT will branch (incorrectly) to any other LUT input pin, eliminating such
+            // stubs.
+            // This feature -- despite intra-site routing updates being deferred -- enables a stub-free physical
+            // netlist to be output. In addition, when reading this netlist back into RapidWright using PhysNetlistReader,
+            // any simulated branches will be discarded allowing deferred updates to continue as before.
```

Comment:
```suggestion
            // Enabling this following option allows the PhysNetlistWriter to simulate LUT pin swapping such that
            // any routes that service an incorrect LUT input site pin are allowed to have
            // a fake branch to the correct site pin on the same LUT for the
            // purposes of simulating (prior to applying deferred updates) a fully
            // routed net.
            // This feature -- despite intra-site routing updates being deferred -- enables a stub-free physical
            // netlist to be output. In addition, when reading this netlist back into RapidWright using PhysNetlistReader,
            // any simulated branches will be discarded allowing deferred updates to continue as before.
```

+            // Ask PhysNetlistWriter to simulate LUT pin swapping such that any
+            // routes that service an incorrect LUT input site pin are allowed to have
+            // a fake branch to the correct site pin on the same LUT for the
+            // purposes of simulating (prior to applying deferred updates) a fully
+            // routed net.
+            // Such fake branches are discarded by CheckPhysNetlist and does not
+            // affect the deferred update process.